tables/vpc_flow_log/vpc_flow_log_extractor.go

```diff
@@ -3,21 +3,29 @@ package vpc_flow_log
 import (
 	"context"
 	"fmt"
-	"strconv"
+	"log/slog"
+	"slices"
 	"strings"
-	"time"
 
 	"github.com/turbot/tailpipe-plugin-sdk/artifact_source"
+	"github.com/turbot/tailpipe-plugin-sdk/formats"
 )
 
+var allLogFields = []string{
+	"account-id", "action", "az-id", "bytes", "dstaddr", "dstport", "end", "flow-direction", "instance-id", "interface-id", "log-status", "packets", "pkt-dst-aws-service", "pkt-dstaddr", "pkt-src-aws-service", "pkt-srcaddr", "protocol", "region", "reject-reason", "srcaddr", "srcport", "start", "sublocation-id", "sublocation-type", "subnet-id", "tcp-flags", "traffic-path", "type", "version", "vpc-id", "ecs-cluster-name", "ecs-cluster-arn", "ecs-container-instance-id", "ecs-container-instance-arn", "ecs-service-name", "ecs-task-definition-arn", "ecs-task-id", "ecs-task-arn", "ecs-container-id", "ecs-second-container-id",
+}
```

Comment:
I feel like this is duplication of a list we already needed - these are essentially the `keys` from the map returned by `getValidTokensAndColumnNames()` which we use to map these values (keys) to our desired column names (values) 

Perhaps you could utilise the keys from this map as opposed to maintaining 2 lists?

Reply:
Yup, updated the code to use `getValidTokensAndColumnNames()` instead.